### ISSUE TITLE
[New build - DEFAULT] DEFAULT moves to 21w05c

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#3603]
Default moves to the latest build!
Cheers all

**Comment by Sebateses** [comment]
Hey Tsuna, Abba circuits don't seem to be working in this build.

**Comment by RockefellerDoctor** [comment]
> 
> 
> Hey Tsuna, Abba circuits don't seem to be working in this build.

Can also confirm, none of my ABBA circuits in my current build work anymore, even after rebuilding them from scratch.

**Comment by Briaireous** [comment]
Indeed, my traffic light logic for the intersection street module fails to send signals between some parts of the logic

**Comment by jealousdequan** [comment]
I can also confirm that my complicated elevator logic is also no longer functional. I've been told by others that it is most likely because of the fact that there is an issue with buttons not activating any logic. Whether thats the case or not, I'm not sure, But it makes sense to me.

**Comment by RockefellerDoctor** [comment]
> 
> 
> I can also confirm that my complicated elevator logic is also no longer functional. I've been told by others that it is most likely because of the fact that there is an issue with buttons not activating any logic. Whether thats the case or not, I'm not sure, But it makes sense to me.

I've tested several ships from the workshop (eg.; ones i didn't build myself) and many of their devices either don't work properly now, or don't work at all.

**Comment by tsunamayo** [comment]
Okay this issue is actually fixed for 21w05c. I will push that patch now.

**Comment by Sebateses** [comment]
@tsunamayo Thankyou :)

**Comment by tsunamayo** [comment]
yep this is the issue https://github.com/tsunamayo/Starship-EVO/issues/3595, I should have done a hotfix build before doing the default merge... Build is uploading now.

**Comment by tsunamayo** [comment]
Okay the new patch is up!
